docs/runtimes/mlrun_jobs.ipynb

```diff
@@ -12,7 +12,7 @@
     "- [Define a new function and its dependencies](#define-a-new-function-and-its-dependencies)\n",
     "- [Convert the code to a serverless job](#convert-the-Code-to-a-serverless-job)\n",
     "- [Deploy (build) the function container](#deploy-build-the-function-container)\n",
-    "- [Run the function on the cCluster](#run-the-function-on-the-cluster)\n",
+    "- [Run the function on the Cluster](#run-the-function-on-the-cluster)\n",
```

Comment:
Run the function on the cluster
Lower case!

Reply:
Good catch, fixed